examples/web-service/index.js

```diff
@@ -34,7 +34,7 @@ app.use('/api', function(req, res, next){
   if (!key) return next(error(400, 'api key required'));
 
   // key is invalid
-  if (!~apiKeys.indexOf(key)) return next(error(401, 'invalid api key'));
+  if (apiKeys.indexOf(key)<0) return next(error(401, 'invalid api key'));
```

Comment:
I personally would use `=== -1` here since `-1` is the value returned by `indexOf` if no match is found?

```suggestion
  if (apiKeys.indexOf(key) === -1) return next(error(401, 'invalid api key'));
```